plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java

```diff
@@ -65,6 +66,11 @@ public JdbcMetadataSessionProperties(JdbcMetadataConfig jdbcMetadataConfig, @Max
                         "Enable TopN pushdown",
                         jdbcMetadataConfig.isTopNPushdownEnabled(),
                         false))
+                .add(integerProperty(
```

Comment:
Add validation here as well:

```suggestion
                .add(integerProperty(
                        INSERT_BATCH_SIZE,
                        "Insert batch size",
                        jdbcMetadataConfig.getInsertBatchSize(),
                        value -> true, // validation logic here
                        false))
```

Reply:
done